src/components/Table/DelegatesTable.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import useResponsive from 'decentraland-gatsby/dist/hooks/useResponsive'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+import { Close } from 'decentraland-ui/dist/components/Close/Close'
+import { Modal } from 'decentraland-ui/dist/components/Modal/Modal'
+import { Table } from 'decentraland-ui/dist/components/Table/Table'
+import Responsive from 'semantic-ui-react/dist/commonjs/addons/Responsive'
+
+import { Delegate } from '../../hooks/useDelegatesInfo'
+import { useSortingByKey } from '../../hooks/useSortingByKey'
+import useVotingPowerInformation from '../../hooks/useVotingPowerInformation'
+import Candidates from '../../modules/delegates/candidates.json'
+import Sort from '../Icon/Sort'
+import VotingPowerDelegationDetail from '../Modal/VotingPowerDelegationDetail/VotingPowerDelegationDetail'
+import { Candidate } from '../Modal/VotingPowerDelegationModal/VotingPowerDelegationModal'
+import DelegateRow from '../Table/DelegatesTableRow'
+
+import './DelegatesTable.css'
+
+const DISPLAYED_CANDIDATES = 9
+const DEFAULT_SORTING_KEY: DelegateKey = 'lastVoted'
+type DelegateKey = keyof Delegate
+
+interface Props {
+  delegates: Delegate[]
+  userAddress: string | null
+}
+
+const DelegatesTable = ({ delegates, userAddress }: Props) => {
+  const t = useFormatMessage()
+  const responsive = useResponsive()
+  const isMobile = responsive({ maxWidth: Responsive.onlyMobile.maxWidth })
+  const [sortingKey, setSortingKey] = useState(DEFAULT_SORTING_KEY as DelegateKey)
```

Comment:
I think you can do here `useState<DelegateKey>(DEFAULT_SORTIG_KEY)` instead of casting the initial state.